feature-detects/audio/preload.js

```diff
@@ -1,13 +1,76 @@
 /*!
 {
-  "name": "Audio Preload Attribute",
+  "name": "Audio Preload",
   "property": "audiopreload",
-  "tags": ["audio", "media"]
+  "tags": ["audio", "media"],
+  "async" : true,
+  "warnings": ["This test is very large – only include it if you absolutely need it"],
+  "knownBugs": ["Not consistent in Safari and IE when run in Cow tests"]
```

Comment:
Not sure if this is necessary to mention that our test suite could give the incorrect results. Seems like we may need to fix that.